engine/swarm/how-swarm-mode-works/pki.md

```diff
@@ -1,43 +1,42 @@
 ---
 description: How PKI works in swarm mode
-keywords: docker, container, cluster, swarm mode, node, tls, pki
-title: How PKI works in swarm mode
+keywords: swarm, security, tls, pki,
+title: Manage swarm security with public key infrastructure (PKI)
 ---
 
-The swarm mode public key infrastructure (PKI) system built into Docker Engine
+The swarm mode public key infrastructure (PKI) system built into Docker
 makes it simple to securely deploy a container orchestration system. The nodes
 in a swarm use mutual Transport Layer Security (TLS) to authenticate, authorize,
-and encrypt the communications between themselves and other nodes in the swarm.
+and encrypt the communications with other nodes in the swarm.
 
-When you create a swarm by running `docker swarm init`, the Docker Engine
-designates itself as a manager node. By default, the manager node generates
-itself a new root Certificate Authority (CA) along with a key pair to secure
-communications with other nodes that join the swarm. If you prefer, you can pass
-the `--external-ca` flag to specify a root CA external to the swarm. Refer to
-the [docker swarm init](../../reference/commandline/swarm_init.md) CLI
-reference.
+When you create a swarm by running `docker swarm init`, Docker designates itself
+as a manager node. By default, the manager node generates a new root Certificate
+Authority (CA) along with a key pair, which are used to secure communications
+with other nodes that join the swarm. If you prefer, you can specify your own
+externally-generated root CA, using the the `--external-ca` flag of the
+[docker swarm init](/engine/reference/commandline/swarm_init.md) command.
 
 The manager node also generates two tokens to use when you join additional nodes
-to the swarm: one worker token and one manager token. Each token includes the
-digest of the root CA's certificate and a randomly generated secret. When a node
-joins the swarm, it uses the digest to validate the root CA certificate from the
-remote manager. It uses the secret to ensure the node is an approved node.
+to the swarm: one **worker token** and one **manager token**. Each token
+includes the digest of the root CA's certificate and a randomly generated
+secret. When a node joins the swarm, it uses the digest to validate the root CA
```

Comment:
Maybe " When a node joins the swarm, the joining node uses the digest to validate the root CA certificate from the remote manager.  The remote manager uses the secret to ensure the joining node is an approved node."?